chore: reducing ILP writer idle timeout defaults (#1063)
questdb/questdb.io: docs/reference/api/ilp/tcp-receiver.md

line.tcp.commit.interval.fraction=0.2
```

> If the fraction value and/or commit lag is set to `0`, the

Reviewer: Would this be more concise?

```suggestion
If the result of commit interval is `0`, the
```
